templates/simple_register_lookup.html

```diff
@@ -73,5 +73,51 @@ <h3>Embed this table</h3>
         <script src="/static/js/simple_register_lookup_core.js" type="text/javascript"></script>
         <script src="/static/js/simple_register_lookup.js" type="text/javascript"></script>
         <script src="/static/js/test_site.js" type="text/javascript"></script>
+         <script type="text/javascript">
+        $( document ).ready(function() {
+ 
+        var urlParams = new URLSearchParams(window.location.search);
+        var Device = urlParams.get('Device'); 
+        var DevicesTag = urlParams.get('Tag'); 
+        var TagsExpand = urlParams.get('Expand'); 
+        var deviceDropdown = document.getElementById('device-dropdown');
+        var len = deviceDropdown.options.length;
+        for(i = 0; i < len; i++)
+         {
+         if (deviceDropdown.options[i].innerHTML == Device)
+         {
+         deviceDropdown.selectedIndex = i;
+         break;
+         }     
+        }
+        var tagdropdown = document.getElementById('tag-dropdown');
+        var len = tagdropdown.options.length;
+        for(i = 0; i < len; i++)
+         {
+         if (tagdropdown.options[i].innerHTML == DevicesTag)
+         {
+         tagdropdown.selectedIndex = i;
+         break;
+         }     
+        }
+        if(TagsExpand == "true"){
+            document.getElementById("expand-checkbox").checked = true;
+        }
+        });
+         $('#device-dropdown').on('change', function() {
+            deeplink();
+        });
+          $('#tag-dropdown').on('change', function() {
+            deeplink();
+        });
+           $('#expand-checkbox').on('change', function() {
+            deeplink();
+        });
+            function deeplink(){
+            var newurl = window.location.protocol + "//" + window.location.host + window.location.pathname + "?Device=" + $("#device-dropdown").val() + "&Tag=" + $("#tag-dropdown").val() + "&Expand=" + $("#expand-checkbox").is(':checked')
+            window.history.pushState({ path: newurl }, '', newurl);
+            }
+
+        </script>
```

Comment:
To stick with convention and for cleanliness, please put this into a new .js file in static/js/.

Reply:
Done